src/server/mod.rs

```diff
@@ -264,36 +250,37 @@ where
     }
 
     /// Start the server and listen for connections on the given address.
+    /// Blocks execution of current thread.
```

Comment:
To me it looks like that this function doesn't start the server and listens, it just returns a future that does that. Also I don't think that this function is blocking since the ```TcpListener``` is from tokoi

Reply:
fixed